msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -1,34 +1,36 @@
 ---
-title: Camera and image capabilities in Teams
-description: How to use Teams Javascript client SDK to enable native camera and image capabilities
-keywords: camera image capabilities native device permissions
+title: Media capabilities in Teams
+description: How to use Teams Javascript client SDK to enable media capabilities
+keywords: camera image microphone capabilities native device permissions media
 ms.topic: conceptual
 ms.author: lajanuar
 ---
 
-# Camera and image capabilities in Teams
+# Media capabilities in Teams
```

Comment:
Should the title be integrate Media capabilities in Teams or Device capabilities. This needs more clarity. In the overview page we need to clearly differentiate between the two terms.